src/front-end/src/components/MentorAccepted.js

```diff
@@ -47,7 +47,7 @@ class MentorAccepted extends Component {
           return (
             <MenteeDisplay
               name={mentee.name}
-              email={mentee.email}
+              email=""
```

Comment:
This is great work, but it requires a slight modification.
On behalf of Codebar, the client requested that Mentee emails be removed from the Mentee requests (so that the email field is not on the cards within the 'Suggested Mentees' tab. Instead, Mentee's emails can be visible on the card component once a Mentor has accepted a Mentee Request [i.e emails can be visible in the 'Accepted Mentees tab']). In short: just apply the reverse to the current setup.
Please let me know if the way that I've described this is confusing, and I can try to describe things in a different way!

Reply:
Yea you are right I should do it in reverse!